pkgs/applications/misc/wlclock/default.nix

```
@@ -0,0 +1,46 @@
+{ pkgs
+, stdenv
+, lib
```

```suggestion
{ lib
, stdenv
```
`pkgs` isn't used and lib should always be first

---

Thanks! I totally missed that xD